.changeset/tricky-files-speak.md

```diff
@@ -0,0 +1,6 @@
+---
+"app-builder-lib": patch
+"builder-util": patch
```

Comment:
Since the configuration schema is removing `includeSubNodeModules`, this needs to be a major semver bump

Reply:
updated